settings.gradle.kts

```diff
@@ -1,10 +1,12 @@
+import org.apache.tools.ant.taskdefs.condition.Os
```

Comment:
what?

Reply:
can detect current OS and is the proper way of doing it in gradle scripts afaik, but was a remnant anyways